float32[] camera_K
float32[] camera_RT
shape_msgs/Plane[] planes_bg
rl_msgs/Graph graph
rl_msgs/SegmentationObject[] objects